techhive-ui/src/components/Internships/Internships.css

```diff
@@ -2,40 +2,42 @@
     padding: 20px;
     font-family: montserrat;
   }
-  
+
   .internship-cards {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
-    gap: 20px;
+    grid-template-columns: 1fr; /* Change this to 1fr */
```

Comment:
why do we need this comment?

Reply:
I was reminding myself to change it, I forgot to take it out